test/new-e2e/tests/agent-metrics-logs/log-agent/k8s-logs/file_tailing_test.go

```diff
@@ -83,12 +88,13 @@ func (v *k8sSuite) TestSingleLogAndMetadata() {
 	}, 1*time.Minute, 10*time.Second)
 }
 
+//go:embed long_line_log.txt
+var longLineLog string
```

Comment:
Note for reviewers: this removes potential errors from `os.ReadFile`, with one assert less